docs/topics/ref_known-issues-7-2-0.adoc

```diff
@@ -30,3 +30,6 @@ Running analysis of the application in containerless mode fails to fetch some de
 
 The report generated by containerless CLI analysis is inconsistent when it has analyzed two binary applications that are located in the same folder. This issue is planned to be resolved in {ProductShortName} 7.2.1. link:https://issues.redhat.com/browse/MTA-4484[(MTA-4484)]
 
+.Duplicate entries are appearing because some files inside the target folder are being analyzed
+
+You can see duplicate entries in the built-in provider searches if the application is compiled using the `mvn clean` package. This issue is planned to be resolved in a later release of {ProductShortName}.
```

Comment:
using the `mvn clean package`

Reply:
You can link here to https://issues.redhat.com/browse/MTA-4260 Jira story.
(I've changed this from a doc bug into a feature story).